starlite/handlers/http.py

```diff
@@ -239,6 +253,22 @@ def resolve_after_request(self) -> Optional[AfterRequestHandler]:
                 self.resolved_after_request = self.resolved_after_request.__func__
         return cast(Optional[AfterRequestHandler], self.resolved_after_request)
 
+    def resolve_exception_handlers(self) -> Dict[Union[int, Type[Exception]], ExceptionHandler]:
+        """
+        Resolves the exception_handlers by starting from the route handler and moving up.
+
+        This method is memoized so the computation occurs only once.
+        """
+        if self.resolved_exception_handlers is BaseRouteHandler.empty:
+            exception_handlers: Dict[Union[int, Type[Exception]], ExceptionHandler] = {}
+            for layer in self.ownership_layers():
```

Comment:
If you reverse the ownership layers you can skip the checks for whether the key exists or not as each layer will overwrite the previous, e.g.:

```python
...
for layer in reversed(self.ownership_layers):
    exception_handlers.update(layer.exception_handlers or {})
self.resolved_exception_handlers = exception_handlers
...
```